### ISSUE TITLE
Create playlist?

### ISSUE THREAD

**Issue by Mirko2297** [arichorn/uYouPlus-Downloads#5]
Can i create playlist in uyou lab? all download song is only in one tab

**Comment by arichorn** [comment]
I don’t think so. I recommend suggesting it at https://github.com/MiRO92/uYou-for-YouTube/issues
